### PR TITLE
User info display

### LINKED ISSUE #571: [Feat]: Fetch User Info onClick
As an EmbeddedChat developer

**I need to**: Integrate a feature to display the current user information whose avatar is clicked.

**So That**: I is rendered as a sidebar and toggled also.

**Acceptance Criteria**

- [x] Integrate relevant API from API docs to get the user info of a particular user.
- [x] Develop and display the information using <Sidebar> component.

**Screenshots**
Screenshot 2024-04-14 212424

### PR DESCRIPTION
**Pull request by umangutkarsh**
# Brief Title
This PR integrates the feature to display the information of a user on clicking the avatar.

## Acceptance Criteria fulfillment

- [X] Integrate relevant API from API docs to get the user info of a particular user.
- [X] Develop and display the information using `<Sidebar>` component.

Fixes #571 

## Video/Screenshots
[userInfo.webm](https://github.com/RocketChat/EmbeddedChat/assets/95426993/acdc4dd0-6068-4091-b14b-a3f23385f143)

*This PR will require a re-build of the api package using -> `yarn build` under  `/packages/api` *

### CODE REVIEW

**Comment by umangutkarsh** [comment]
Hi @sidmohanty11. Could you please review the PR?

File: packages/react/src/components/UserInformation/UserInformation.js [hunk 1]
```diff
@@ -0,0 +1,308 @@
...
+  const authenticatedUserId = useUserStore((state) => state.userId);
+  const isAdmin = authenticatedUserRoles.includes('admin');
+
+  const getUserAvatarUrl = (username) => {
+    const host = RCInstance.getHost();
+    const URL = `${host}/avatar/${username}`;
+    return URL;
+  };
+
+  const formatTimestamp = (timestamp) => {
```
Comment (sidmohanty11) [inline packages/react/src/components/UserInformation/UserInformation.js:51]: can we please move this to a separate file inside https://github.com/RocketChat/EmbeddedChat/tree/develop/packages/react/src/lib?

**Review by sidmohanty11 — COMMENTED** [review]
LGTM! Just left a minor comment

File: packages/react/src/components/UserInformation/UserInformation.js (hunk 1 above)
Comment (umangutkarsh) [reply packages/react/src/components/UserInformation/UserInformation.js]: @sidmohanty11 
Modified the code as required.

**Pull request merged**